### PR TITLE
Rename FileSystemSnapshotVisitor.{visit -> visitFile}

### PR DESCRIPTION
**Pull request by wolfs**
So it is more obvious that directory snapshots are not passed into this
method.

### CODE REVIEW

File: subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/FileSystemSnapshotVisitor.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
     /**
      * Called for each regular/missing file.
      */
-    void visit(FileSystemLocationSnapshot fileSnapshot);
+    void visitFile(FileSystemLocationSnapshot fileSnapshot);
```
Comment (lptr) [inline subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/FileSystemSnapshotVisitor.java]: I think the Javadoc needs to be updated to include unavailable files.

Comment (wolfs) [reply subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/FileSystemSnapshotVisitor.java]: Done.

**Comment by wolfs** [comment]
@lptr PTAL!

File: subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/FileSystemSnapshotVisitor.java (hunk 1 above)
Comment (lptr) [reply subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/FileSystemSnapshotVisitor.java]: And I guess now it has to be reverted? :o

Comment (wolfs) [reply subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/FileSystemSnapshotVisitor.java]: Well, it still will receive the unavailable files, the file type of those will be missing, though...

I am fine either way.

Comment (lptr) [reply subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/FileSystemSnapshotVisitor.java]: Me too. Let's leave it as it is.

**Pull request merged**